var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -47,9 +47,12 @@ class Legion(CMakePackage):
     version('17.02.0', '31ac3004e2fb0996764362d2b6f6844a')
 
     variant('debug', default=False, description='Build debug version')
+    variant('mpi', default=False,
+            description='Build on top of mpi conduit for mpi inoperability')
```

Comment:
My only concern here is whether or not `mpi` should default to True. There's a lot of bugs in the concretizer right now that make it hard for package A to default to `+mpi` and its dependency package B to default to `~mpi`. In #2590 I made sure that every `mpi` variant defaulted to True to workaround this until the concretizer is fixed. Would you be opposed to having the default be True?